susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
+if [ -z ${PRODUCT+x} ];then
+    PRODUCT="Uyuni"
+fi
+
+while getopts 'P:h' c
+do
+    case $c in
+        P) PRODUCT=$OPTARG ;;
+        h) echo "Usage $0 [-p PRODUCT]";exit -2;;
+    esac
+done
 
 HERE=`dirname $0`
-. $HERE/VERSION
+
+if [ ! -f $HERE/VERSION.${PRODUCT} ];then
```

Comment:
```suggestion
if [ ! -f ${HERE}/${VPRODUCT} ];then
```

Reply:
Enclising variable names with `{}` is always a good practice, as then you can remember to do it when the variable is inside texts.